include/mbedtls/check_config.h

```diff
@@ -673,6 +673,14 @@
 #error "MBEDTLS_HAVE_INT32/MBEDTLS_HAVE_INT64 and MBEDTLS_HAVE_ASM cannot be defined simultaneously"
 #endif /* (MBEDTLS_HAVE_INT32 || MBEDTLS_HAVE_INT64) && MBEDTLS_HAVE_ASM */
 
+#if defined(MBEDTLS_FS_IO_ALT) && ( !defined(MBEDTLS_FS_IO) )
+#error "MBEDTLS_FS_IO_ALT defined, but not all prerequisites"
+#endif
+
```

Comment:
Does it make sense to have `MBEDTLS_SERIALIZE_C` but not `MBEDTLS_FS_IO`? As far as I understand, `MBEDTLS_SERIALIZE_C` enables a serialization-based implementation of the FS abstraction.

Reply:
Serialisation can be used for `MBEDTLS_NET_C` code as well.